flink/v2.0/flink/src/test/java/org/apache/iceberg/flink/sink/TestIcebergSink.java

```diff
@@ -421,8 +421,10 @@ void testOperatorsUidNameWitUidSuffix() throws Exception {
 
   @TestTemplate
   void testErrorOnNullForRequiredField() throws Exception {
-    Assume.assumeFalse(
-        "ORC file format supports null values even for required fields.", format == FileFormat.ORC);
+
+    assumeThat(format)
+            .as("ORC file format supports null values even for required fields.")
+            .isEqualTo(FileFormat.ORC);
```

Comment:
The current one uses `assumeFalse` to omit tests for ORC, so use `isNotEqualTo`, not `isEqualTo`

```suggestion
    assumeThat(format).isNotEqualTo(FileFormat.ORC);
```

Reply:
If you have a chance, could you apply this when you send the commit to this PR along with running the spotlessApply?

Reply:
Sure, I’ll include that as well!

Reply:
> If you have a chance, could you apply this when you send the commit to this PR along with running the spotlessApply?

I have revised the PR by incorporating your feedback.
Thank you for your valuable advice!

#13021